src/main/resources/blackDuckPlugin.properties

```diff
@@ -69,5 +69,16 @@ blackduck.artifactory.inspect.policy.repos=
 blackduck.artifactory.inspect.policy.repos.csv.path=
 blackduck.artifactory.inspect.policy.severity.types=BLOCKER,CRITICAL,MAJOR,MINOR,TRIVIAL,UNSPECIFIED
 #
+# Scan-as-a-Service (scaas)
+# If Scan-as-a-Service is enabled, Scanner and Inspector *should* be disabled
```

Comment:
Do we enforce shutoff of the other services or is this truly up to the customer?

Reply:
I will change the comment...Originally, the plan was to not forcefully turn off the other modules if the ScanAsAService module was enabled. That changed at the last minute...see lines 94-101 in https://github.com/blackducksoftware/blackduck-artifactory/pull/57/files#diff-a950af77cd2eb6011339956a0b39b6ceb3a04a1aab255efe6c7d493dc3371a1d